default/stringtables/en.txt

```diff
@@ -13968,21 +13968,21 @@ System Defense Mines 1
 DEF_SYST_DEF_MINE_1_DESC
 '''When friendly ships are not present in a system, enemy ships are free to blockade the system, stopping the transfer of resources without directly engaging the planet's defenses, thus with no risk of damage to themselves. A network of system defense mines can deter these tactics by damaging enemy ships each turn they are in the system.
 
-The [[metertype METER_STRUCTURE]] of all ships belonging to an enemy empire which are present in a system containing a planet owned by the empire that possesses this technology will be reduced at a rate of 16 per turn.
+The [[metertype METER_STRUCTURE]] of all ships belonging to an enemy empire which are present in a system containing a planet owned by the empire that possesses this technology will be reduced at a rate of [[value SYSTEM_MINES_1]] per turn.
 
 Mines regenerate each turn, so are not depleted by use.'''
 
 DEF_SYST_DEF_MINE_2
 System Defense Mines 2
 
 DEF_SYST_DEF_MINE_2_DESC
-Reduces the [[metertype METER_STRUCTURE]] of enemy ships by an additional 32 per turn, cumulative with [[tech DEF_SYST_DEF_MINE_1]].
+Reduces the [[metertype METER_STRUCTURE]] of enemy ships a total of [[value SYSTEM_MINES_2]] per turn.
```

Comment:
you changed the description from "cumulative" to "a total of" but don't appear to change the scripts. Was the text out of date before this, or is it still cumulative?

Reply:
The macros themselves are based on a priority system by my reading.

// This macro handles multiple entries, where only the earliest applicable effects a target.

The old stringtable entries were fixed values rather then in-game.

Reply:
should probably make the descriptions clear about whether the effects are cumulative or not, regardless of how that's implemented...

Reply:
do you mean, "a total of, not cumulative with.." or something similar?

Reply:
yes. "a total of" is probably better than "not cumulative with", though it's misleading if there can be something else that overrides it.